src/components/Timer.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { displayMilliseconds } from '../utils/index';
+
+export class Timer extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            time: new Date().getTime(),
+        };
+    }
+    componentDidMount() {
+        this.timerInterval = setInterval(
+            () => this.tick(),
+            1000,
+        );
+    }
+    componentWillUnmount() {
+        clearInterval(this.timerInterval);
+    }
+    tick() {
+        this.setState({
+            time: new Date().getTime(),
+        });
+    }
+    // 86400000 = 24 hour * 60 min * 60s * 1000ms
+    render() {
+        return (
+            <span>
+                {this.props.symbolOpenTime.symbolOpenTime ? displayMilliseconds(86400000 - (this.state.time - this.props.symbolOpenTime.symbolOpenTime)) : '--'}
```

Comment:
@nazaninreihani this algorithm just works when the open time is in 24 hours. 
@ashkanx Do we have any cases that the open time is after the next 24 hours?